README.md

```diff
@@ -63,6 +61,28 @@ Then, run the following command:
 $ pod install
 ```
 
+### Carthage
+
+[Carthage](https://github.com/Carthage/Carthage) is a simple, decentralized dependency manager for Cocoa.
+
+```bash
+brew install carthage
+```
+
+To integrate the `omisego` SDK into your Xcode project using Carthage, add the following line in your `Cartfile`:
+
+```
+github "omisego/ios-sdk"
+```
+
+Then, run the following command:
+
+```bash
+carthage update --platform ios
+```
+
+Drag `OmiseGo` in the project.
```

Comment:
I'd put: 
Drag the built `OmiseGO.framework` into your Xcode project.
For maximum clarity :)